gateway/compose.production.yaml

```diff
@@ -300,5 +301,39 @@ services:
     #         - sds-gateway-prod-minio-net
     #         - sds-network-prod
 
+    jupyter:
+        # JupyterHub for interactive data analysis and notebooks (PRODUCTION)
+        build:
+            context: .
+            dockerfile: ./compose/production/jupyter/Dockerfile
+            args:
+                JUPYTERHUB_VERSION: latest
+        restart: always
+        image: sds-gateway-prod-jupyter
+        container_name: sds-gateway-prod-jupyter
+        networks:
+            - sds-network-prod
```

Comment:
maybe let's put jupyter in a separate subnet / create a network with only the containers that need to share it